src/lexer/jsx.ts

```diff
@@ -96,8 +97,9 @@ export function scanJSXToken(parser: ParserState, context: Context): Token {
         if (CharTypes[parser.currentChar] & CharFlags.JSXToken) break;
       }
 
-      parser.tokenValue = parser.source.slice(parser.tokenPos, parser.index);
-      if (context & Context.OptionsRaw) parser.tokenRaw = parser.tokenValue;
+      const raw = parser.source.slice(parser.tokenPos, parser.index);
+      if (context & Context.OptionsRaw) parser.tokenRaw = raw;
+      parser.tokenValue = decodeHTMLStrict(raw);
```

Comment:
I wonder how this impact performance, maybe use a `getter`?

Reply:
easier to hoist `const { tokenValue, tokenPos, fisker, tokenRaw } = parser` and re-use

Reply:
Not sure, changing to getter might touch lots of code. @KFlash?